Rakefile

```diff
@@ -302,8 +324,33 @@ We use information from your github account throughout the class.
         verify: -> { assert(!github_username.to_s.empty?, "We can't find your github username.", "") }
       },
 
+      global_gitignore: {
+        header: "Register a global gitignore file",
+        installation_steps: [
+          %q(
+1. Backup your existing global_gitignore (if it exists).  You can ignore a "No such file or directory" error:
```

Comment:
Good suggestion, but difficult to provide.  These instructions are also available as a markdown file.    Options:
- add additional step (with instructions and verify) to backup existing file (if it exists).
- We could take care of this in the installfest.sh.  It would just put the gitignore file in place and not configure git.  Felt like too much magic - difficult line to balance here.
- skip the backup, but we got slapped for stomping on configurations in the first few cohorts, that's why we use instead of the GA script.